code/modules/mob/living/simple_animal/guardian/guardiannaming.dm

```diff
@@ -1,103 +1,169 @@
-
+#define GUARDIANCOLOROVERLAY "color overlay" //Used with spritecolor var for guardians to determine what color to apply
+#define GUARDIANCOLORBASE "color base"
+#define GUARDIANCOLORBOTH "color both"
+#define GUARDIANCOLORNONE "color none"
 /datum/guardianname
 	var/prefixname = "Default" //the prefix the guardian uses for its name
 	var/suffixcolour = "Name" //the suffix the guardian uses for its name
-	var/parasiteicon = "techbase" //the icon of the guardian
+	var/parasitebaseicon = "techbase" //the icon of the guardian
+	var/parasiteoverlayicon = "techglowbase"
```

Comment:
KAAAAAAAAAAAAAAAZE
this will make parasites look like absolute dongs; it's a sprite quality downgrade because it needs four overlays to replicate how it looks currently(and four color vars for each of those colors); this just applies one color to the entire thing, and not even the RIGHT color.